packages/atomic/src/components/search/atomic-search-interface/atomic-search-interface.ts

```diff
@@ -83,273 +109,284 @@ export class AtomicSearchInterface
    * <atomic-search-interface fields-to-include='["fieldA", "fieldB"]'></atomic-search-interface>
    * ```
    */
-  @ArrayProp()
-  @Prop({mutable: true})
-  public fieldsToInclude: string[] | string = '[]';
+  @property({
```

Comment:
As discussed live, we currently do not do prop validation / re-validation on any interface (i.e., commerce, insight, search)... This is something we should address (not necessarily in this PR, though).

Ideally, we'd have a reactive controller to always do it the same way in every component.